package.json

```
@@ -2,27 +2,25 @@
   "name": "design.yousee.dk",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "",
+  "engines": {
+    "node": "^10"
+  },
   "scripts": {
     "start": "gulp",
-    "build": "gulp fractalTheme && gulp fractal:build",
     "lint:scss": "gulp lint-scss"
   },
   "repository": {},
-  "author": "Shift ApS",
+  "author": "YouSee Digital Warehouse",
   "license": "ISC",
-  "homepage": "",
+  "homepage": "https://github.com/youseedk/ysds",
```

```suggestion
  "homepage": "https://projectname.yousee.dk",
```

Og så smider vi det andet ned under repository:
```suggestion
"repository": {
    "type": "git",
    "url": "git+https://github.com/youseedk/ysds"
  },
  "bugs": {
    "url": "https://github.com/youseedk/ysds/issues"
  },
```